gossip-base/src/test/java/org/apache/gossip/crdt/LWWSetTest.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
MeanGirls has this definition:

{
  'type': 'lww-e-set',
  'bias': 'a',
  'e': [
    ['a', 0],
    ['b', 1, 2],
    ['c', 2, 1],
    ['d', 3, 3]
  ]
}

a was created at 0 and still exists.
b was deleted after creation; it does not exist.
c was created after deletion; it exists
d was created and deleted at the same time. Bias a means we prefer adds, so it exists.

Try to implement these cases directly as one or more tests